engine/src/module-canvas.cpp

```diff
@@ -5642,6 +5669,8 @@ void MCCanvasErrorsFinalize()
 	MCValueRelease(kMCCanvasGradientStopRangeErrorTypeInfo);
 	MCValueRelease(kMCCanvasGradientStopOrderErrorTypeInfo);
 	MCValueRelease(kMCCanvasGradientTypeErrorTypeInfo);
+
+	MCValueRelease(kMCSVGPathParseErrorTypeInfo);
```

Comment:
This should be in the Canvas namespace - i.e. kMCCanvasSVGPathParseErrorTypeInfo.